src/registers.h

```diff
@@ -0,0 +1,150 @@
+#include <iostream>
+#include <stdint.h>
+using namespace std;
+
+/*
+Header file for register implementation.
+
+Each block contains two 8-bit registers, then a simulated 16-bit register 
+created by joining the 8-bit registers together.
+
+There is a getter and setter for each register. The getters and setters for 
+the combined registers get and set based on what is contained in the 8-bit registers.
+
+There are additionally two naturally 16-bit registers, pc and sp.
+*/
```

Comment:
Oh quick tip here, if this is formatted like this, then most IDEs will automatically shows this text if you hover over the class name (also works for functions and the like)
```cpp
/**
 * Description.......
 */
```

Reply:
okay, sound's good. I'll reformat the block comments to fit this style